plugins/inputs/docker_log/README.md

```diff
@@ -0,0 +1,66 @@
+# Docker Log Input Plugin
+
+The docker log plugin uses the Docker Engine API to get logs on running
+docker containers.
+
+The docker plugin uses the [Official Docker Client](https://github.com/moby/moby/tree/master/client)
+to gather logs from the [Engine API](https://docs.docker.com/engine/api/v1.24/).
+
+### Configuration:
+
+```toml
+# Read metrics about docker containers
+[[inputs.docker_log]]
+  ## Docker Endpoint
+  ##   To use TCP, set endpoint = "tcp://[ip]:[port]"
+  ##   To use environment variables (ie, docker-machine), set endpoint = "ENV"
+  endpoint = "unix:///var/run/docker.sock"
+
+  ## Only collect metrics for these containers. Values will be appended to
+  ## container_name_include.
+  ## Deprecated (1.4.0), use container_name_include
+  container_names = []
```

Comment:
Remove this config option as there isn't much value in replicating deprecated options from the `docker` plugin.

Reply:
Taken care of this!